tests/php/Control/UserDefinedFormControllerTest.php

```diff
@@ -580,4 +581,28 @@ public function testParseByteSizeStringBadValuesThrowException($input)
         $controller = new SizeStringTestableController(); // extends UserDefinedFormController
         $controller->convertSizeStringToBytes($input);
     }
+
+    /**
+     * Test that provided email is valid
+     */
+    public function testValidEmailsToArray()
+    {
+        $class = new ReflectionClass(UserDefinedFormController::class);
+        $method = $class->getMethod('validEmailsToArray');
+        $method->setAccessible(true);
+
+        $controller = new UserDefinedFormController();
+
+        $this->assertEquals([], $method->invokeArgs($controller, [null]));
```

Comment:
Sorry should have mentioned first time around, could you change this to use `@dataProvider provideValidEmailsToArray` and move the fixtures to a new method `provideValidEmailsToArray` - this is the standard way that we've adopted for keeping unit tests nice and tidy - search around the codebase for `@dataProvider` and you should find some examples of this being done

Reply:
DONE